utils/rpms/daos.spec

```diff
@@ -554,9 +554,12 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
-* Fri Apr  8 2022 Dmitry Eremin <dmitry.eremin@intel.com> 2.3.100-2
+* Fri Apr  8 2022 Dmitry Eremin <dmitry.eremin@intel.com> 2.3.100-3
```

Comment:
You need to update https://github.com/daos-stack/daos/blob/c24625a9d5a07b790d8335a8423d787f13717fb2/utils/rpms/daos.spec#L30 from `2` to `3` also.

Reply:
Due to Wei Li's PR just landed, I bumped release to 4.

Reply:
Sorry for the merge conflict. #8608 was fixing a serious correctness issue and the head of a long dependency queue.

Reply:
No problem at all. :)